framework/src/interfaces/OrientedBoxInterface.C

```diff
@@ -13,7 +13,7 @@
 #include "InputParameters.h"
 #include "MooseError.h"
 
-// libMesh incluces
+#include "libmesh/auto_ptr.h"
```

Comment:
Not sure why this is here. Did you use it in a downstream object?

Reply:
There is a `libmesh_make_unique` call here in this file. That macro is defined in `auto_ptr.h`.